src/lib/components/InputChip/InputChip.svelte

```diff
@@ -0,0 +1,116 @@
+<script lang="ts">
+	import { createEventDispatcher } from 'svelte/internal';
+
+	// Event Dispatcher
+	const dispatch = createEventDispatcher();
+
+	// Props (settings)
+	/** Optionally provide an semantic label. */
+	export let label: string = '';
+	/** Optionally provide an input placeholder. */
+	export let value: string[] = [];
+	/**
+	 * Provide a whitelist of accepted values.
+	 * @type {string[]}
+	 */
+	export let whitelist: string[] = [];
+	/** When enabled, will format entered values as lowercase. */
+	export let lowercase: boolean = true;
+	/**
+	 * Provide a custom validation function.
+	 * @type {function}
+	 */
+	export let validation: any = undefined;
+
+	// Props (styles)
+	/** Provide chip styles. */
+	export let chip: string = `chip-primary`;
+	/** Provide chip hover styles. */
+	export let hover: string = 'hover:chip-error';
+	/** Set the input padding styles. */
+	export let padding: string = 'p-3';
+	/** Set the input border radius styles. */
+	export let rounded: string = 'rounded-container-token';
+	/** Set the input focus ring styles. */
+	export let ring: string = 'focus:ring-transparent';
+
+	// Local
+	let inputValue: string = '';
+	let inputInvalid: boolean = false;
+
+	// Classes
+	const cBase = 'unstyled border-token flex flex-wrap gap-2 items-center';
+	const cLabel = 'unstyled w-full md:w-auto mr-2';
+	const cInput = 'unstyled flex-auto border-transparent bg-transparent text-base px-1 py-0 focus:border-transparent min-h-[30px]';
+
+	function resetValiadtionClass(): void {
+		inputInvalid = false;
+	}
+
+	function addChip(event: any): void {
+		event.preventDefault();
+		// Validate: custom validation
+		if (validation !== undefined && !validation(inputValue)) {
+			inputInvalid = true;
```

Comment:
My 2c would be to refactor as `inputIsValid`; comparing to truthy is always easier, the cognitive load for "objectIsNotOk is true" is a little harder on the smoothbrains like me